cmd/amplifier/main.go

```diff
@@ -18,6 +18,8 @@ var (
 	elasticsearchDefaultURL = "http://localhost:9200"
 	kafkaDefaultURL         = "localhost:9092"
 	influxDefaultURL        = "http://localhost:8086"
+	dockerDefaultURL        = "unix:///var/run/docker.sock"
```

Comment:
Don't need to have `///` if you use backticks for [raw string literals](https://golang.org/ref/spec#String_literals) instead of normal quotes:

    dockerDefaultURL = `unix://var/run/docker.sock`